entries/o/outlook.com.json

```
@@ -1,17 +1,23 @@
 {
   "Outlook.com": {
     "domain": "outlook.com",
+    "additional-domains": [
+      "outlook.live.com"
```

Won't hurt adding hotmail as well. There's probably some edge case where they still use that domain :roll_eyes: 
```suggestion
      "outlook.live.com",
      "hotmail.com"
```

---

"hotmail"... That's a name I haven't heard in a while.

Do we really want `live.com` here? This could clash with "OneDrive" (`onedrive.live.com`).

---

I didn't realize they used the live.com domain for anything else. The suggestion is updated now.

Hotmail is still very much alive and sadly being used. Their servers on the other hand are very much [**not** alive](https://postmaster.live.com).